sass/index.scss

```diff
@@ -1,166 +1,632 @@
+// Variables
+// Colors
+$dark-blue: #193787;
+$blue: #1447C8;
+$light-blue: #44C6E6;
+$off-white: #F6F8FA;
+$black: #081646;
```

Comment:
I may be colorblind, but...
<img width="838" alt="CleanShot 2021-11-24 at 22 32 09@2x" src="https://user-images.githubusercontent.com/2782749/143375137-f19305e7-4d6f-42b7-a655-2175387f32d4.png">

Reply:
Per Figma, we are mostly using this color to replace the normal #000000 black of a text color
![image](https://user-images.githubusercontent.com/56453920/143392478-55909bf0-f4dd-4286-811d-c4779dd78713.png)
darker-blue and darkest-blue didn't make a lot of sense to me for differentiating variable names
and on club site i was told off-black was an odd color name despite its parallel to off-white

Reply:
Gotcha; yeah I personally would lean towards `darker-blue` if our convention is to stick by `var names == color`, but switching to functionality here `background-blue` or something could work, but risk inconsistency, but `$black != black` sticks in my head more

Reply:
changing it to `darker-blue`